packages/paste-website/src/pages/components/textarea/index.mdx

```diff
@@ -65,11 +65,11 @@ export const getStaticProps = async () => {
 
 ## Guidelines
 
-A textarea allows users to enter text on multiple lines. It is used with the [Label](/components/label) and [Help Text](/components/help-text) components.
+A Textarea allows users to enter text on multiple lines. It is used with the [Label](/components/label) and [Help Text](/components/help-text) components.
 
 ### About Textarea
 
-Use a textarea when you want to allow a user to enter text on multiple lines. A textarea allows for any text to be entered and can't be restricted as other [inputs](/components/input/) can.
+Use a Textarea when you want to allow a user to enter text on multiple lines. A Textarea allows for any text to be entered and can't be restricted as other [inputs](/components/input/) can.
 
 ### Accessibility
 
```

Comment:
suggestion: Similar comment as above. This section could be replaced with markdown so the Help Text and inline error text links are easier to fix.